{
  return new Runnable(){
    @Override public void run(){
      try {
        port.call();
      }
 catch (      Exception e) {
        port.die();
        if (e instanceof IOException) {
          m_logger.trace("VoltPort died, probably of natural causes",e);
        }
 else {
          e.printStackTrace();
          networkLog.error("VoltPort died due to an unexpected exception",e);
        }
      }
 finally {
        addToChangeList(port);
      }
    }
  }
;
}
